src/Data/List.purs

```diff
@@ -584,40 +584,36 @@ span _ xs = { init: Nil, rest: xs }
 
 -- | Group equal, consecutive elements of a list into lists.
 -- |
--- | For example,
--- |
```

Comment:
I think it's pretty obvious that what follows is an example, so figured it would be better to remove that line to make the pursuit results more compact.